util/src/test/scala/stryker4s/scalatest/LogMatchers.scala

```diff
@@ -2,10 +2,26 @@ package stryker4s.scalatest
 
 import ch.qos.logback.classic.Level
 import ch.qos.logback.classic.spi.ILoggingEvent
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.matchers.{BeMatcher, MatchResult}
-import stryker4s.TestAppender
+import stryker4s.{Stryker4sSuite, TestAppender}
 
-trait LogMatchers {
+trait LogMatchers extends BeforeAndAfterEach {
```

Comment:
Not a fan of having to extend `BeforeAndAfterEach` in the `LogMatchers`  if you need it then maybe place it inside the `Stryker4sSuite` because now you will sometimes have before and after each available if you do log testing. In my opinion that makes no sense, it makes more sense to always have it inside the `Stryker4sSuite` in this case.